starter/README.md

```diff
@@ -6,7 +6,115 @@ This is the project template for Stark applications.
+|   |   |   crossdomain.xml             # cross domain policies
+|   |   |   favicon.ico                 # icon for bookmarks bar
+|   |   |   humans.txt                  # contains information about the website (http://humanstxt.org/)
+|   |   |   manifest.json               # application icons for Android mobile devices
+|   |   |   README.md                   # 
+|   |   |   robots.txt                  # the robots exclusion protocol (http://www.robotstxt.org/)
+|   |   \   service-worker.js           # support for building Progressive Web Applications (PWA) with Service Workers 
+|   |
+|   +---environments                    # configuration variables for each environment
+|   |   |                               # 
+|   |   |   environment.e2e.prod.ts     # e2e tests configuration
+|   |   |   environment.hmr.ts          # development with HMR (Hot Module Replacement) configuration
+|   |   |   environment.prod.ts         # production configuration
+|   |   |   environment.ts              # development configuration
+|   |   \   model.ts                    # interface defining the variables
+|   |
+|   +---styles                          # application-specific styles
+|   |   \
+|   |
+|   |   custom-typings.d.ts             # add your own type definitions that can't be found in the registry in this file
+|   |   hmr.ts                          # Hot Module Replacement logic
+|   |   index.html                      # the main HTML page that is served when someone visits this site
+|   |   main.browser.ts                 # Angular bootstrapping
+|   |   polyfills.browser.ts            # this file includes polyfills needed by Angular and is loaded before the app
+|   |   stark-app-config.json           # Stark configuration
+|   \   stark-app-metadata.json         # Application metadata like: name, description, version...
+|   
+|   .dockerignore                       # files and directories to excluded by the Docker build
+|   .gitignore                          # files and directories to excluded by git
+|   .prettierignore                     # files and directories to excluded by prettier
+|   .prettierrc.js                      # exports the stark-build/config/.prettierrc.js file
+|   .stylelintrc                        # extends the stark-build/config/stylelint.config.js
+|   .travis.yml                         # YAML file to customize the Travis build (https://travis-ci.org/)
+|   angular.json                        # Angular configuration file
+|   base.spec.ts                        # Initializes the test environment
+|   Dockerfile                          # the commands that will be executed by the Docker Build command
+|   karma.conf.ci.js                    # exports the Karma configuration from stark-testing/karma.conf.ci.js
+|   karma.conf.js                       # exports the Karma configuration from stark-testing/karma.conf.js
+|   package.json                        # 
+|   package-lock.json                   # 
```

Comment:
I think we can remove this file from the list since it is not part of the project, I mean the code in the repo. This is automatically generated by NPM after running 'npm install'